lib/pbench/server/api/__init__.py

```diff
@@ -71,7 +71,7 @@ def register_endpoints(api, app, config):
     )
     api.add_resource(
         DatasetsDetail,
-        f"{base_uri}/datasets/detail",
+        f"{base_uri}/datasets/detail/<string:dataset_name>",
```

Comment:
I'd like to keep these names consistent. I already used `dataset` for `/datasets/metadata`, and I don't see any value in the longer name.